### PR TITLE
Ensuring target_file_size is an int to prevent a calculation error [hotfix] [ENG-2385]

### PR DESCRIPTION
**Pull request by corbinSanders**
## Purpose

Sometimes a data provider gives us back a string size instead of an int size. We need the int.

## Changes

Ensure the size metadata is an int

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify no errors are caused when moving files between google drive and osf storage. This is where the error occured

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised? N/A

## Documentation

N/A
## Side Effects

Shouldn't be.

## Ticket

https://openscience.atlassian.net/browse/ENG-2385

### CODE REVIEW

File: api/caching/tasks.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def update_storage_usage_with_size(payload):
     provider = metadata.get('provider', 'osfstorage')
 
     target_file_id = metadata['path'].replace('/', '')
-    target_file_size = metadata.get('size', 0)
+    target_file_size = int(metadata.get('size', 0))
```
Comment (jwalz) [inline api/caching/tasks.py:161]: drive-by: should we trust the 'sizeInt' that Water Butler already gives us instead?

Comment (corbinSanders) [reply api/caching/tasks.py:161]: I suppose reading the whole ticket is beneficial! 👍

**Review by jwalz — APPROVED** [review]

**Review by brianjgeiger — APPROVED** [review]
